test/test_acceptance_tests.py

```diff
@@ -0,0 +1,172 @@
+import io
+import sys
+import unittest
+from unittest.mock import patch
+from src.game import Game
+from test.mocks.mock_message import MockMessage
+
+
+class TestAcceptance(unittest.TestCase):
+    @patch("builtins.input", side_effect=["1", "1", "2", "3", "4", "5", "6", "7", "n"])
+    def test_player_X_wins_the_game(self, mock_input):
+        captured_output = io.StringIO()
+        game = Game(message=MockMessage())
+        expected_message = MockMessage.declare_winner(self, "X")
+
+        sys.stdout = captured_output
+        game.run()
+        sys.stdout = sys.__stdout__
+
+        self.assertIn(expected_message, captured_output.getvalue())
```

Comment:
there is a pretty cool pattern going on here.  Could you extract a method and remove the duplication?

Reply:
Hmmmm....it's possible!  gonna need to think on that...

Reply:
I think @tpetersen0308 have the same [idea](https://github.com/t-keazirian/tic-tac-toe/pull/25#pullrequestreview-955554854)